packages/mutation-testing-elements/src/components/mutation-test-report-mutant/index.scss

```diff
@@ -4,3 +4,7 @@
 .disabled-code {
   text-decoration: line-through;
 }
+
+span.badge {
+    text-shadow: none;
```

Comment:
different formatting used. Apparently we're missing a formatter?

Reply:
I can add Prettier in a different PR if you like? (Maybe after some of the other open PR's are merged 😃)